src/gui/styles/base/BaseStyle.cpp

```diff
@@ -1936,7 +1936,7 @@ void BaseStyle::drawPrimitive(PrimitiveElement elem,
         auto fropt = qstyleoption_cast<const QStyleOptionFocusRect*>(option);
         if (!fropt)
             break;
-        //### check for d->alt_down
+        // ### check for d->alt_down
```

Comment:
intended change?

Reply:
This was done by the auto formatter.

Reply:
Huh. Resetting the change and re-running the formatter did not result in this change again.